assets/style.scss

```diff
@@ -0,0 +1,86 @@
+@import "../node_modules/bootstrap/scss/functions";
+
+$white: #fff;
+
+$spacer: 1rem;
+
+$spacers: (
+  0: 0,
+  1: $spacer * 0.25,
+  2: $spacer * 0.5,
+  3: $spacer,
+  4: $spacer * 1.5,
+  5: $spacer * 3,
+  6: $spacer * 3.75,
+);
+
+$blue: #306fca;
+$secondary: #87b2f0;
+$body-color: #304b72;
+$link-color: $secondary;
+
+$link-decoration: none;
+
+$paragraph-margin-bottom: 20px;
+
+$h1-font-size: 30px;
+$h2-font-size: 24px;
+$h3-font-size: 22px;
+$h4-font-size: 18px;
+
+$headings-margin-bottom: 20px;
+$headings-font-family: "Alata", sans-serif;
+
+$text-muted: #979da6;
+
+$input-btn-focus-width: 0;
+
+$navbar-nav-link-padding-x: 29px;
+
+$navbar-dark-color: $white;
+
+$navbar-dark-toggler-border-color: none;
+
+@import "../node_modules/bootstrap/scss/variables";
+
+@import "../node_modules/bootstrap/scss/mixins";
+@import "../node_modules/bootstrap/scss/utilities";
+
+// Layout & components
+@import "../node_modules/bootstrap/scss/root";
+@import "../node_modules/bootstrap/scss/reboot";
+@import "../node_modules/bootstrap/scss/type";
+@import "../node_modules/bootstrap/scss/images";
+@import "../node_modules/bootstrap/scss/containers";
+@import "../node_modules/bootstrap/scss/grid";
+@import "../node_modules/bootstrap/scss/tables";
+@import "../node_modules/bootstrap/scss/forms";
+@import "../node_modules/bootstrap/scss/buttons";
+@import "../node_modules/bootstrap/scss/transitions";
+@import "../node_modules/bootstrap/scss/dropdown";
+@import "../node_modules/bootstrap/scss/button-group";
+@import "../node_modules/bootstrap/scss/nav";
```

Comment:
Is this a generated file?

Reply:
No, I copied this from the initial project in order to keep the same styling. I think the new generated page is pixel-by-pixel identical (minus fixed bugs)